integration/grpc_test.go

```diff
@@ -28,6 +29,28 @@ func TestAppendTokenToOutgoingContext(t *testing.T) {
 	}
 }
 
+func ExampleAppendTokenToOutgoingContext() {
+	// someFunc doesn't do anything, but in a real-world situation it might
+	// send a request to some grpc service that requires an authorization
+	// token
+	someFunc := func(context.Context) {
+		// send a request to some grpc service
```

Comment:
you should print the contents of the token here to test that the context is properly decorated, then verify it with the `//Output:...` comment at the end of the func